ocaml-lsp-server/test/e2e/package.json

```diff
@@ -8,22 +8,18 @@
     "promote": "jest --updateSnapshot"
   },
   "dependencies": {
-    "jest-cli": "26.6.3",
     "outdent": "0.8.0",
     "vscode-uri": "^3.0.2"
   },
   "devDependencies": {
-    "@types/jest": "26.0.20",
+    "@types/jest": "27.0.2",
     "@types/node": "14.14.25",
-    "jest": "26.6.3",
+    "jest": "27.3.1",
     "prettier": "2.2.1",
-    "ts-jest": "26.5.1",
+    "ts-jest": "27.0.7",
     "typescript": "4.1.3",
     "vscode-jsonrpc": "6.0.0",
     "vscode-languageserver-protocol": "3.16.0",
     "vscode-languageserver-types": "3.16.0"
-  },
-  "jest": {
-    "preset": "ts-jest"
```

Comment:
Fixes that warning:

```
● Multiple configurations found:
    * /Users/corentin/Dev/ocaml-lsp/ocaml-lsp-server/test/e2e/jest.config.js
    * `jest` key in /Users/corentin/Dev/ocaml-lsp/ocaml-lsp-server/test/e2e/package.json

  Implicit config resolution does not allow multiple configuration files.
  Either remove unused config files or select one explicitly with `--config`.

  Configuration Documentation:
  https://jestjs.io/docs/configuration.html
```

This config has been moved in the `jest.config.js` file